src/framework/shortcuts/data/shortcuts.xml

```diff
@@ -794,7 +794,7 @@
     </SC>
   <SC>
     <key>zoomin</key>
-    <seq>Ctrl++</seq>
+    <seq>Ctrl+=</seq>
```

Comment:
I don't think that it is a good idea to change the shortcut in this way. We should find out why Ctrl++ doesn't work properly

Reply:
To get to = I'd have to press Shift, on a German QWERTZ keyboard... whereas the + is accessible without Shift on this keyboard. So replacing one nuisance with another...

Reply:
Well, I think when the Shortcut was written as `Ctrl++`, it implies that plus also has to be pressed. But there is no direct plus key. You have to press `Shift + =` to get a plus. Hence, when `Ctrl + +` implies `Ctrl + Shift + =`. Now, when you click `Ctrl + Shift + =` , it does imply Zoom in, hence it work while `Ctrl + =` does not

Reply:
If you need Shift to get to =, so be it. Doesn't have any bearing to the shoutcut itself. Or at least shouldn't (and didn't in 3.x)
Same is true for many other shortcuts where the keys differ depending on zhe keyboard usedd, like {[]}-_.:,;+* and many more

Reply:
I'm also curious, do you have to press `Ctrl + Shift + =` to zoom in your browser because of your keyboard?

Reply:
No, as + is accessible without Shift on my keyboard, it is Ctrl++ here.

Reply:
Oh it seems like there are two shortcuts, I just checked I can zoom in with `Ctrl + Shift + =` (`Ctrl + +`) as well

Reply:
And s `Ctrl`+`+` just works for me in the current builds too.

Reply:
Yeah, but the "universal" zoom in button, in my experience, is `Ctrl + =` (So no need to press shift too). That's why I tried fixing this

Reply:
Well, no, it is not. The universal key is `Ctrl`+`+` for zooming in and  `Ctrl`+`-` for zooming out. As said, whether  you have to press `Shift` to get to either is irrelevant and depending on keyboard layout